lib/blackout_generator.rb

```diff
@@ -0,0 +1,29 @@
+module BlackoutGenerator
+  def generate
+    Blackout.create! do |blk|
+      blk.start_date = time_rand(Time.zone.now, Time.zone.now + 1.year)
+      blk.end_date = time_rand(blk.start_date, blk.start_date.next_week)
+      blk.notice = FFaker::HipsterIpsum.paragraph(2)
+      blk.created_by = User.first.id
+      blk.blackout_type = %w(soft hard).sample
+    end
+  end
+
+  private
+
+  def time_rand(start, finish)
+    range = (start..finish)
+    random_time = rand(range)
+    blackouts = Blackout.all.map { |blk| blk.start_date..blk.end_date }
```

Comment:
Actually, should it even be its own module then? I didn't move any of the other generators that only had a single method